kubejs/server_scripts/expert/recipes/ars_nouveau/enchanting_apparatus.js

```diff
@@ -0,0 +1,136 @@
+ServerEvents.recipes((event) => {
+    if (global.isExpertMode == false) {
+        return;
+    }
+
+    const id_prefix = 'enigmatica:expert/ars_nouveau/enchanting_apparatus/';
+    const recipes = [
+        {
+            output: { item: 'create:mechanical_crafter', count: 8 },
+            pedestalItems: [
+                { item: { item: 'create:brass_casing' } },
```

Comment:
`{ item: { item: 'create:brass_casing' } }` - is that the correct format?

Reply:
Yeah :-\